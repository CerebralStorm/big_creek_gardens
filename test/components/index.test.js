import React from 'react'
import Index from '../../app/javascript/components/index'

test('render a Index', () => {
  const wrapper = shallow(<Index />);
  expect(wrapper).toMatchSnapshot();
});