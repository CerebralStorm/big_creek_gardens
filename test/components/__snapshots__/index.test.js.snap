// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a Index 1`] = `
<div>
  <section
    className="no-margin"
    id="main-slider"
  >
    <div
      className="carousel slide"
      data-ride="carousel"
      id="carouselControls"
    >
      <div
        className="carousel-inner"
      >
        <div
          className="carousel-item active"
        >
          <img
            alt="First slide"
            className="d-block w-100"
            src="bg1.jpg"
          />
          <div
            className="carousel-caption d-none d-md-block"
          >
            <div
              className="carousel-content"
            >
              <h2
                className="animation animated-item-1"
              >
                <img
                  className="img-fluid"
                  src="logo-alpha.png"
                />
              </h2>
              <p
                className="animation animated-item-2"
              >
                Hemp products that are out of this world!
              </p>
              <Link
                className="btn-slide animation animated-item-3"
                replace={false}
                to="/shop"
              >
                Shop Now!
              </Link>
            </div>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <img
            alt="Second slide"
            className="d-block w-100"
            src="bg2.jpg"
          />
          <div
            className="carousel-caption d-none d-md-block"
          >
            <div
              className="carousel-content"
            >
              <h2
                className="animation animated-item-1"
              >
                <img
                  className="img-fluid"
                  src="logo-alpha.png"
                />
              </h2>
              <p
                className="animation animated-item-2"
              >
                Hemp products that are out of this world!
              </p>
              <Link
                className="btn-slide animation animated-item-3"
                replace={false}
                to="/shop"
              >
                Shop Now!
              </Link>
            </div>
          </div>
        </div>
        <div
          className="carousel-item"
        >
          <img
            alt="Third slide"
            className="d-block w-100"
            src="bg3.jpg"
          />
          <div
            className="carousel-caption d-none d-md-block"
          >
            <div
              className="carousel-content"
            >
              <h2
                className="animation animated-item-1"
              >
                <img
                  className="img-fluid"
                  src="logo-alpha.png"
                />
              </h2>
              <p
                className="animation animated-item-2"
              >
                Hemp products that are out of this world!
              </p>
              <Link
                className="btn-slide animation animated-item-3"
                replace={false}
                to="/shop"
              >
                Shop Now!
              </Link>
            </div>
          </div>
        </div>
      </div>
      <a
        className="carousel-control-prev"
        data-slide="prev"
        href="#carouselControls"
        role="button"
      >
        <span
          aria-hidden="true"
          className="carousel-control-prev-icon"
        />
        <span
          className="sr-only"
        >
          Previous
        </span>
      </a>
      <a
        className="carousel-control-next"
        data-slide="next"
        href="#carouselControls"
        role="button"
      >
        <span
          aria-hidden="true"
          className="carousel-control-next-icon"
        />
        <span
          className="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </section>
  <div
    className="feature"
  >
    <div
      className="container"
    >
      <div
        className="text-center row"
      >
        <div
          className="col-md-3"
        >
          <h2>
            Full Responsive
          </h2>
          <p>
            Quisque eu ante at tortor imperdiet gravida nec sed turpis phasellus.
          </p>
        </div>
        <div
          className="col-md-3"
        >
          <h2>
            Retina Ready
          </h2>
          <p>
            Quisque eu ante at tortor imperdiet gravida nec sed turpis phasellus.
          </p>
        </div>
        <div
          className="col-md-3"
        >
          <h2>
            Full Responsive
          </h2>
          <p>
            Quisque eu ante at tortor imperdiet gravida nec sed turpis phasellus.
          </p>
        </div>
        <div
          className="col-md-3"
        >
          <h2>
            Friendly Code
          </h2>
          <p>
            Quisque eu ante at tortor imperdiet gravida nec sed turpis phasellus.
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="about"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col"
        >
          <h2>
            about us
          </h2>
          <img
            className="img-fluid"
            src="6.jpg"
          />
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus interdum erat libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque
          </p>
        </div>
        <div
          className="col"
        >
          <h2>
            Template built with Twitter Bootstrap
          </h2>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus interdum erat libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus interdum erat libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus interdum erat libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="lates"
  >
    <div
      className="container"
    >
      <div
        className="text-center"
      >
        <h2>
          Lates News
        </h2>
      </div>
      <div
        className="row"
      >
        <div
          className="col-md-4"
        >
          <img
            className="img-fluid"
            src="4.jpg"
          />
          <h3>
            Template built with Twitter Bootstrap
          </h3>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus interdum erat libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque
          </p>
        </div>
        <div
          className="col-md-4"
        >
          <img
            className="img-fluid"
            src="4.jpg"
          />
          <h3>
            Template built with Twitter Bootstrap
          </h3>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus interdum erat libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque
          </p>
        </div>
        <div
          className="col-md-4"
        >
          <img
            className="img-fluid"
            src="4.jpg"
          />
          <h3>
            Template built with Twitter Bootstrap
          </h3>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus interdum erat libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque libero, pulvinar tincidunt leo consectetur eget. Curabitur lacinia pellentesque
          </p>
        </div>
      </div>
    </div>
  </div>
  <section
    id="partner"
  >
    <div
      className="container"
    >
      <div
        className="center wow fadeInDown"
      >
        <h2>
          Our Partners
        </h2>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut 
          <br />
           et dolore magna aliqua. Ut enim ad minim veniam
        </p>
      </div>
      <div
        className="partners"
      >
        <ul>
          <li>
             
            <a
              href="#"
            >
              <img
                className="img-fluid"
                src="partner1.png"
              />
            </a>
          </li>
          <li>
             
            <a
              href="#"
            >
              <img
                className="img-fluid"
                src="partner2.png"
              />
            </a>
          </li>
          <li>
             
            <a
              href="#"
            >
              <img
                className="img-fluid"
                src="partner3.png"
              />
            </a>
          </li>
          <li>
             
            <a
              href="#"
            >
              <img
                className="img-fluid"
                src="partner4.png"
              />
            </a>
          </li>
          <li>
             
            <a
              href="#"
            >
              <img
                className="img-fluid"
                src="partner5.png"
              />
            </a>
          </li>
        </ul>
      </div>
    </div>
  </section>
</div>
`;
